<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Filter App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply Filter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="filterLabel">
      <property name="text">
       <string>Select Filter:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="filterComboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Inversion</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brightness Correction</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Contrast Enhancement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gamma Correction</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaussian Blur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sharpen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Edge Detection</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Emboss</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset Image</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QGridLayout" name="imagesGrid">
      <item row="0" column="0">
       <widget class="QLabel" name="originalImageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="filteredImageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
